### PR TITLE
Create how-to-enable-a-mailbox.md

### PR DESCRIPTION
**Pull request by hikarutilmitt**

### CODE REVIEW

**Comment by nimbinatus** [comment]
@hikarutilmitt , in the last step, is the **Select Action** part a dropdown?

**Comment by nimbinatus** [comment]
Opened PR on the patch-22 branch, hikarutilmitt/rackspace-how-to#1

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-92e036410f/how-to/)

File: content/cloud-sites/how-to-enable-a-mailbox.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
```
Comment (DonNateR) [inline content/cloud-sites/how-to-enable-a-mailbox.md]: Do you know how to delete an article? This is a duplicate.

**Pull request merged**